docs/control-structures.rst

```diff
@@ -125,7 +125,7 @@ throws an exception or goes out of gas.
 .. note::
     Before Solidity 0.6.2, the recommended way to specify the value and gas
     was to use ``f.value(x).gas(g)()``. This is still possible but deprecated
```

Comment:
It's no longer possible, is it :-)?

Reply:
I'm fixing this now, this isn't complete yet.